addon/components/toggle-buttons.js

```diff
@@ -13,7 +13,6 @@ export default class ToggleButtons extends Component {
     if (! this.args.firstOptionSelected) {
       this.args.toggle(true);
     }
-    console.log(this.elementId);
```

Comment:
wheeee!

Reply:
😆 forgot that was in there. I was super confused why my console was spitting out these weird `undefined` messages. Probably need to add an ESLint rule for that!